src/patternfly/components/Page/page.scss

```diff
@@ -154,13 +154,19 @@ $pf-c-page--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl",
   @media screen and (min-width: $pf-global--breakpoint--xl) {
     --pf-c-page__main-breadcrumb--PaddingRight: var(--pf-c-page__main-breadcrumb--xl--PaddingRight);
     --pf-c-page__main-breadcrumb--PaddingLeft: var(--pf-c-page__main-breadcrumb--xl--PaddingLeft);
+    --pf-c-page__header--c-masthead--inset: var(--pf-c-page__header--c-masthead--xl--inset);
   }
 
   // Wizard main section
   --pf-c-page__main-wizard--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
   --pf-c-page__main-wizard--BorderTopColor: var(--pf-global--BorderColor--100);
   --pf-c-page__main-wizard--BorderTopWidth: var(--pf-global--BorderWidth--sm);
 
+  // masthead
+  --pf-c-page__header--c-masthead--inset: var(--pf-global--spacer--md);
+  --pf-c-page__header--c-masthead--xl--inset: var(--pf-global--spacer--lg);
+  --pf-c-page__header--c-masthead-toggle--c-button--FontSize: var(--pf-global--FontSize--2xl);
```

Comment:
should this just be the default? It seems reasonable as a default, most mobile toggles are pretty big. Or is there a reason we want the default to be something different? I would expect the masthead's default behavior to work like it should in the page component, but I may be unaware of other use cases for it.